packages/web-forms/src/components/controls/Input/InputNumeric.vue

```diff
@@ -0,0 +1,211 @@
+<script setup lang="ts">
+import type { InputNumberInputEvent } from 'primevue/inputnumber';
+import PrimeInputNumber from 'primevue/inputnumber';
+import type { Ref } from 'vue';
+import { computed, customRef, inject, ref } from 'vue';
+
+interface NumericNodeState {
+	get required(): boolean;
+	get readonly(): boolean;
+}
+
+interface NumericNodeAppearances {
+	readonly 'thousands-sep'?: boolean;
+}
+
+interface NumericNode {
+	readonly nodeId: string;
+	readonly currentState: NumericNodeState;
+	readonly appearances: NumericNodeAppearances;
+}
+
+interface InputNumericProps {
+	readonly node: NumericNode;
+	readonly numericValue: number | null;
+	readonly setNumericValue: (value: number | null) => void;
+	readonly isDecimal?: boolean;
+	readonly min?: number;
+	readonly max?: number;
+}
+
+const props = defineProps<InputNumericProps>();
+
+interface FractionalDigitOptions {
+	readonly min?: number;
+	readonly max?: number;
+}
+
+let fractionalDigits: FractionalDigitOptions;
+
+/**
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode}
+ */
+type NumericInputMode = 'decimal' | 'numeric';
+
+let inputMode: NumericInputMode;
+
+if (props.isDecimal) {
+	fractionalDigits = {
+		min: 0,
+		max: 18,
```

Comment:
I see this is minimum conformance defined by XML. Collect caps at 15 total characters (including the decimal). Let's track this difference for possible follow-up work.